package.json

```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "@material-ui/core": "^1.1.0",
     "@material-ui/icons": "^1.1.0",
-    "ajv": "^6.5.0",
+    "@reactivex/rxjs": "^6.2.0",
+    "ajv": "^5.5.2",
```

Comment:
I'm not complaining but is there a reason why you chose `ajv` instead of `Joi`?

Reply:
i think this came from "Layouts" branch when we merge branches to get full view. Maybe its for typography elements or something, not sure https://github.com/re-pe/simul-doc-fe/pull/9/commits/d363f58ba0a44a9078e8b7c65e004a9e6407fc98